android/src/main/kotlin/com/hiennv/flutter_callkit_incoming/CallkitNotificationManager.kt

```diff
@@ -186,7 +186,11 @@ class CallkitNotificationManager(private val context: Context) {
                     .into(targetLoadAvatarDefault)
             }
             val caller = data.getString(CallkitConstants.EXTRA_CALLKIT_NAME_CALLER, "")
-            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
+            val notificationManager: NotificationManager =
+                context.getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager
+            val canUseFullScreenIntent =
+                Build.VERSION.SDK_INT < Build.VERSION_CODES.UPSIDE_DOWN_CAKE || notificationManager.canUseFullScreenIntent()
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P && canUseFullScreenIntent) {
```

Comment:
This block seems to set the only CallStyle.
It is not related to full screen.

You should check this here(L125).

https://github.com/AAkira/flutter_callkit_incoming/pull/24/files#diff-0ee07ecfd0bb91b3a7aaf8f1c530fbc1cebaf45b78539854cfd2256446c24d49R125

Reply:
I have reverted this change.

When removing the line `<uses-permission android:name="android.permission.USE_FULL_SCREEN_INTENT"/>` from AndroidManifest.xml, the following error occurred.

```
Not posted. CallStyle notifications must be for a foreground service or user initated job or use a fullScreenIntent.
```

However, I have reverted the USE_FULL_SCREEN_INTENT line in AndroidManifest.xml so this fix is no longer necessary.